app/components/candidate_interface/previous_applications_component.html.erb

```diff
@@ -0,0 +1,27 @@
+<table class="govuk-table govuk-!-margin-top-6">
+  <caption class="govuk-table__caption app-table__caption--m">Your previous applications</caption>
```

Comment:
This can be changed to the govuk-frontend class now, I managed to get it added upstream. 😄 

```suggestion
  <caption class="govuk-table__caption govuk-table__caption--m">Your previous applications</caption>
```

Reply:
👍